gneiss/_summary.py

```diff
@@ -57,3 +58,72 @@ def __init__(self, stat_results,
         # calculate the overall coefficient of determination (i.e. R2)
         sst = sse + ssr
         self.r2 = 1 - sse / sst
+
+    def _check_projection(self, project):
```

Comment:
Are you planning to use this check in other methods? If not, why not add it in the coefficients code (it looks like a really simple check that is happening within `coefficients`)?